src/main/java/org/bahmni/reports/util/SqlUtil.java

```diff
@@ -75,15 +76,21 @@ public static ResultSet executeSqlWithStoredProc(Connection connection, String f
                 }
                 hasMoreResultSets = stmt.getMoreResults(); //true if it is a resultset
             }
-        }catch (SQLSyntaxErrorException e){
+        } catch (SQLSyntaxErrorException e) {
             throw new InvalidConfigurationException("Column that you have configured in sortBy is either not present in output of the report or it is invalid column");
-        }
-        catch (SQLException e) {
+        } catch (SQLException e) {
             e.printStackTrace();
             throw e;
         }
 
         return null;
     }
+
+    public static String escapeSql(String str) {
```

Comment:
if this is about protection against SQL Injection, then need to find a better way to do this. This does not take of hexa or other forms. Check one such implementation [here](https://github.com/Bahmni/bahmni-core/blob/master/bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/util/SqlQueryHelper.java#L85)